### PR TITLE
LF edge renaming readme files

### PR DESCRIPTION
**Pull request by tPeeush**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Supported Versions
 
-Information about different FIDO Device Onboard (FDO) release versions can be found on the [FDO Release page](https://secure-device-onboard.github.io/docs/latest/releases/).
+Information about different FIDO Device Onboard (FDO) release versions can be found on the [FDO Release page](https://fido-device-onboard.github.io/docs/latest/releases/).
```
Comment (SSWAMIN1SSS) [inline SECURITY.md:5]: Instead of docs , please add docs-fidoiot

File: SECURITY.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
 ## Reporting a Vulnerability
```
Comment (SSWAMIN1SSS) [inline SECURITY.md:8]: In Line 9 , the hyperlink is https://wiki.lfedge.org/display/SDO/Reporting+Issues , it has SDO in it can it be modified

**Review by DukeDavis12 — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -179,7 +179,7 @@ Uncomment `subjectAltName` and allowed list of IP and DNS in `[alt_names]` secti
 
 ### Starting Standalone Database for PRI servers
 
-1. Copy generated `secrets/` folder to `<fdo-pri-src>/component-sample/demo/db` folder. [Generate secrets](#Generating-random-passwords-using-keys_gen.sh)
+1. Copy generated `secrets/` folder to `<fdo-pri-src>/component-sample/demo/db` folder. [Generate secrets](https://github.com/fido-device-onboard/pri-fidoiot#generating-random-passwords-using-keys_gensh)
```
Comment (v-vasavi) [inline README.md:182]: fidoiot#generating is there a space needed?

Comment (tPeeush) [reply README.md:182]: > fidoiot#generating is there a space needed?

No.

**Review by v-vasavi — APPROVED** [review]

**Comment by KiranSukhavasi** [comment]
Jenkins CI is not reporting back with CI status.

**Pull request merged**